src/items/itemsTable.component.tsx

```diff
@@ -219,23 +238,57 @@ export function ItemsTable(props: ItemTableProps) {
         Header: TableHeaderOverflowTip,
         accessorFn: (row) => (row.item.is_defective === true ? 'Yes' : 'No'),
         id: 'item.is_defective',
+        filterVariant: COLUMN_FILTER_VARIANTS.boolean,
+        enableColumnFilterModes: false,
         size: 200,
-        filterVariant: 'autocomplete',
       },
       {
         header: 'Usage Status',
         Header: TableHeaderOverflowTip,
         accessorFn: (row) => row.item.usage_status,
         id: 'item.usage_status',
-        size: 200,
-        filterVariant: 'autocomplete',
+        filterVariant: 'multi-select',
+        filterFn: 'arrIncludesSome',
+        columnFilterModeOptions: ['arrIncludesSome', 'arrIncludesNone'],
+        renderColumnFilterModeMenuItems: ({ onSelectFilterMode }) => [
+          <MenuItem
+            key="arrIncludesSome"
+            onClick={() => onSelectFilterMode('arrIncludesSome')}
+          >
+            Includes
+          </MenuItem>,
+          <MenuItem
+            key="arrIncludesNone"
+            onClick={() => onSelectFilterMode('arrIncludesNone')}
+          >
+            Excludes
+          </MenuItem>,
+        ],
+        size: 250,
```

Comment:
Can you increase the size of the usage statuses column to the same as the dates please. Also can look at all the column sizes to make sure that they are appropriate. The purchase order number seems to be too big.

Reply:
Can you also increase the system column as well